lib/blacklight/component.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Blacklight
+  class Component < ViewComponent::Base
+    class << self
```

Comment:
Please add documentation around what this is doing and why.

Reply:
Documentary comments added.